gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,34 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+from astropy.tests.helper import pytest, assert_quantity_allclose
+from astropy.coordinates import Angle
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data, requires_dependency
+from ...utils.energy import EnergyBounds, Energy
+from ...data import DataStore, ObservationList
+
+
+@requires_dependency('scipy')
+@requires_data('gammapy-extra')
+def test_make_psftable():
+    center = SkyCoord(83.63, 22.01, unit='deg')
+    store = gammapy_extra.filename("datasets/hess-crab4-hd-hap-prod2")
+    data_store = DataStore.from_dir(store)
+    obs1 = data_store.obs(23523)
+    obs2 = data_store.obs(23526)
+    energy = EnergyBounds.equal_log_spacing(1, 10, 100, "TeV")
+    energy_band = Energy([energy[0].value, energy[-1].value], energy.unit)
+    psf1 = obs1.make_psf(source_position=center, energy=energy, theta=None)
+    psf2 = obs2.make_psf(source_position=center, energy=energy, theta=None)
+    psf1_int = psf1.table_psf_in_energy_band(energy_band, spectral_index=2.3)
+    psf2_int = psf2.table_psf_in_energy_band(energy_band, spectral_index=2.3)
+    list = [obs1, obs2]
```

Comment:
Replace this:
```
list = [obs1, obs2]
obslist = ObservationList(list)
```
with this:
```
obslist = ObservationList([obs1, obs2])
```
(`list` is a Python built-in type, and you shouldn't overwrite it by assigning `[obs1, obs2]` to it ... that can cause very weird issues further down and it's also something PyCharm or other static code analysers will warn you about not to do.)